scripts/idl/BUILD.gn

```diff
@@ -25,6 +25,7 @@ pw_python_package("idl") {
     "matter_grammar.lark",
 
     # Templates used for generation
+    "generators/cpp/BridgeClustersCpp.jinja",
```

Comment:
Combining a generic `cpp` name for generators and a template that is `BridgeClustersCpp` seems mismatched.

are these cpp generic (in which case the jinja template should have a different name) or are they bridge specific (in which case the generator name should be named differently than cpp to make it clear what it generates.